api/src/repositories/taxonomy-repository.ts

```diff
@@ -53,7 +53,7 @@ export class TaxonomyRepository extends BaseRepository {
    *
    * @param {number} itisTsn
    * @param {string} itisScientificName
-   * @param {(string | null)} commonName
+   * @param {(string | null)} commonNames
```

Comment:
This JS doc change should also be reflected in the actual function

Reply:
I understand the database doesn't support multiple common names. I think a good compromise for now would be to pass an array of common names, then simply store the first item in the array, and assume that anyone calling this function should only expect the first array item to be stored.